api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -101,37 +103,35 @@
 
     /**
      * <p>
-     * The Type element of the LRA annotation indicates whether a bean method
+     * The Type element of the LRA annotation indicates whether a resource method
      * is to be executed within the context of an LRA.
      *
      * <p>
      * If the method is to run in the context of an LRA and the annotated class
-     * also contains methods annotated with {@link Compensate}/{@link Compensate}
-     * then the bean will enlisted with the LRA. Enlisting with an LRA means that
-     * the bean will be notified when the current LRA is later cancelled/closed.
+     * also contains a method annotated with {@link Compensate}
```

Comment:
The resource is also enlisted when only a @Complete annotated method is present.

Reply:
No it isn't, @Compensate is mandatory.

Reply:
Personally I think that both `@Compensate` and `@Complete` should be mandatory - they used to be but one of the Camel team argued that Complete should be optional. When I accepted that change request I added the example text in the above quote.

Reply:
I guess it was @nicolaferraro with whom this discussion took place? If the idea is to change this to make them both mandatory we should try to get some feedback from wider community

Reply:
Yes, I remember I mentioned I've found some use cases where you don't want to be notified when the LRA is @Complete. For example it makes a lot of sense in **passive** services.

E.g. I've a system that just tracks hotel reservations and has operations for booking a room or cancelling the reservation (@Compensate). Since this system is passive, once a room is booked, it does not make any difference if the LRA is completed or not: the room will be unavailable for others. If it receives a call to "compensate", it will free the room. But it won't do anything on @Complete.

Reply:
@nicolaferraro Do you mind if I use that example?

Reply:
@nicolaferraro I included the example but if that was not okay then I will back out the commit.

Reply:
That's great @mmusgrov , thanks. It helps clarifying this point.

Reply:
@mmusgrov Ok. Probably a rare case that you only want to perform some action on @Complete  but not on @Compensate.